frmts/jp2kak/jp2kakdataset.cpp

```diff
@@ -2215,25 +2215,33 @@ JP2KAKCreateCopy( const char * pszFilename, GDALDataset *poSrcDS,
         return nullptr;
     }
 
+    const bool bReversible =
+            CSLFetchNameValue(papszOptions,"Creversible") != nullptr
+            ? CPLFetchBool( papszOptions, "Creversible", false )
+            : (dfQuality < 99.5);
```

Comment:
test in wrong direction
```suggestion
            : (dfQuality >= 99.5);
```

Reply:
Fix in b0bc8297008c5f7899a093e32aa0164377d4c183